pages/partials/navtab.ejs

```diff
@@ -0,0 +1,3 @@
+<li class="nav-item">
+    <a class="nav-link <% if (navSubPage == activeSubPage) { %>active text-primary<% } else { %>text-secondary<% } %>" href="<%= urlPrefix %><%= urlSuffix %>"><i class="<%= iconClasses %>"></i> <%= tabLabel %></a>
```

Comment:
```suggestion
    <a class="nav-link <% if (navSubPage == activeSubPage) { %>active text-dark<% } else { %>text-secondary<% } %>" href="<%= urlPrefix %><%= urlSuffix %>"><i class="<%= iconClasses %>"></i> <%= tabLabel %></a>
```
So I mentioned earlier that I'm not a fan of making the `.active` text blue since it's too much of a <a href="#">**click-me**</a> UX pattern.

In the spirit of mimicking github, what about making it `text-dark` and customizing `border-top-*` via `public/stylesheets/local.css`:
```
.nav-tabs .nav-link.active {
    border-top-color: var(--primary);
    border-top-width: 0.2rem;
}
```
(or ideally using only bootstrap border utils, but I couldn't get it to work since navtabs leverage transparency: https://getbootstrap.com/docs/4.1/utilities/borders/)

Just my 2c. I won't belabor this anymore if @mwest1066 disagrees 😃 

![image](https://user-images.githubusercontent.com/2503508/78506465-43102d00-773f-11ea-8f08-1df09b521b15.png)

Reply:
Very nice! I agree that it's bad to use blue for text that is not a link.

That blue bar at the top looks great. Has it resulted in the text "Preview" shifting downwards slightly? The text baseline looks lower than that in the inactive tabs.

Reply:
I like it a lot too! @mwest1066 the fix would be to add another CSS rule for the inactive tabs that gives them the same tab-border but fully transparent

Reply:
Works for me. I prefer to avoid custom CSS and use vanilla Bootstrap, but this clearly seems like an area where the Bootstrap defaults are not good and we need to do it by hand.

Reply:
Could also switch to `px` and use a negative `margin-top` so we only add 1 custom css rule without modifying inactive tabs:
```
.nav-tabs .nav-link.active {
    border-top-color: var(--primary);
    border-top-width: 3px;
    margin-top: -2px; /* realign text with inactive tabs (give back 1px border-top) */
}
```

Reply:
Look like:

![image](https://user-images.githubusercontent.com/2503508/78507681-3263b500-7747-11ea-918d-e16f0d129c79.png)

Reply:
Extra nice!!